psi4/driver/p4util/exceptions.py

```diff
@@ -97,6 +98,14 @@ def __init__(self, eqn_description, maxit, wfn):
         self.wfn = wfn
 
 
+class SCFConvergenceError(ConvergenceError):
+    """Error called for problems with SCF iterations."""
+
+    def __init__(self, eqn_description, maxit, wfn):
```

Comment:
I would try to put more important args first. So something more like:
`def __init__(self, wfn, wfn_name, maxiter, e_conv, d_conv):`